CondFormats/SiPixelTransient/src/SiPixelGenError.cc

```diff
@@ -754,6 +755,23 @@ int SiPixelGenError::qbin(int id,
   sy1 = (1.f - yratio) * thePixelTemp_[index].enty[ilow].syone + yratio * thePixelTemp_[index].enty[ihigh].syone;
   sy2 = (1.f - yratio) * thePixelTemp_[index].enty[ilow].sytwo + yratio * thePixelTemp_[index].enty[ihigh].sytwo;
 
+  if (irradiationCorrections) {
+    auto yavggen = (1.f - yratio) * thePixelTemp_[index].enty[ilow].yavggen[binq] +
```

Comment:
a comment line may help to be more obvious why this code should be here rather than 40 lines below.
Unless I missed something, the code here is the same as in the old version, but it apparently depends on what's happening with the used values between here and the old place

Reply:
@slava77 the code is indeed the same. 
The problem lies in the fact that "the parameters used in the y-interpolation were changed prior to the x-interpolation.  The minimal change was to move the y-related quantities to before this happens." (quoting Morris here)

Reply:
Staring at the code for a bit I think what is different between here and 40 lines below are the indices "ilow" and "ihigh" which designate which two template entries are being interpolated between. These are changed on what is now line 812. 

These are different between the x and y interpolation and the previous version had the irradiation bias correction for the y-interpolation use the indices for the x-interpolation.